examples/duel/.gitignore

```diff
@@ -1 +1,3 @@
 target/
+node_modules/
```

Comment:
Oh, alright. The other apps just pull use the deps out of the node_modules folder so that's why I asked. :)